static/js/pages/committee-single.js

```diff
@@ -159,20 +159,33 @@ var occupationColumns = [
 
 var filingsColumns = [
   {
-    data: 'pdf_url',
+    data: 'document_description',
     className: 'all',
     orderable: false,
     render: function(data, type, row, meta) {
       var anchor = document.createElement('a');
-      anchor.textContent = 'View filing';
-      anchor.setAttribute('href', data);
+      anchor.textContent = data;
+      anchor.setAttribute('href', row.pdf_url);
       anchor.setAttribute('target', '_blank');
       return anchor.outerHTML;
     }
   },
-  {data: 'amendment_indicator', className: 'min-desktop'},
-  {data: 'report_type_full', className: 'min-desktop'},
+  {
+    data: 'amendment_indicator',
+    className: 'min-desktop',
+    render: function(data, type, row, meta) {
```

Comment:
This function is repeated below. How about putting it in another file (maybe `helpers.js`) and writing a little more concisely:
```javascript
var amendmentMap = {
  A: 'Amended',
  N: 'New'
};
function(data) {
  return amendmentMap[data] || '';
}
```